.travis.yml

```diff
@@ -18,32 +18,37 @@ env:
+   - DESTINATION="OS=8.2,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
+   - DESTINATION="OS=8.3,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
+   - DESTINATION="OS=8.4,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
 
-   - DESTINATION="OS=9.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
-   - DESTINATION="OS=9.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="NO"  POD_LINT="NO"
-   - DESTINATION="OS=9.2,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="NO"  POD_LINT="NO"
-   - DESTINATION="OS=9.3,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="NO"  POD_LINT="NO"
+   - DESTINATION="OS=9.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_IOS_EXAMPLE="YES" BUILD_TVOS_EXAMPLE="NO" POD_LINT="YES"
+   - DESTINATION="OS=9.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
+   - DESTINATION="OS=9.2,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
+   - DESTINATION="OS=9.3,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_IOS_EXAMPLE="NO"  BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
 
-   - DESTINATION="OS=10.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO"
+   - DESTINATION="OS=10.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_IOS_EXAMPLE="YES" BUILD_TVOS_EXAMPLE="NO" POD_LINT="NO"
 
-   - DESTINATION="OS=9.0,name=Apple TV 1080p"  SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO" POD_LINT="NO"
-   - DESTINATION="OS=10.0,name=Apple TV 1080p" SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO" POD_LINT="NO"
+   - DESTINATION="OS=9.0,name=Apple TV 1080p"  SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="YES" BUILD_IOS_EXAMPLE="NO" BUILD_TVOS_EXAMPLE="YES" POD_LINT="NO"
+   - DESTINATION="OS=10.0,name=Apple TV 1080p" SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="YES" BUILD_IOS_EXAMPLE="NO" BUILD_TVOS_EXAMPLE="YES" POD_LINT="NO"
 
 script:
 - if [ $POD_LINT == "YES" ]; then
       bundle exec pod lib lint;
   fi
 
 
-- if [ $BUILD_EXAMPLE == "YES" ]; then
+- if [ $BUILD_IOS_EXAMPLE == "YES" ]; then
       xcodebuild build -workspace Example/IGListKitExamples.xcworkspace -scheme IGListKitExamples -sdk "$SDK" -destination "$DESTINATION" ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO | bundle exec xcpretty -c;
```

Comment:
Let's revert these changes. Instead, inside of this `if` we can check the value of `$SDK`